experimenter/experimenter/experiments/models.py

```diff
@@ -812,35 +822,41 @@ def timeline(self):
                 "label": self.Status.DRAFT,
                 "date": self.draft_date,
                 "is_active": self.is_draft,
+                "days": None,
             },
             {
                 "label": self.Status.PREVIEW,
                 "date": self.preview_date,
                 "is_active": self.is_preview,
+                "days": None,
             },
             {
                 "label": self.PublishStatus.REVIEW,
                 "date": self.review_date,
                 "is_active": self.is_review,
+                "days": None,
             },
             {
-                "label": self.Status.LIVE,
+                "label": NimbusConstants.ENROLLING,
                 "date": self.start_date,
-                "is_active": self.is_live,
+                "is_active": self.is_enrolling,
+                "days": None,
             },
             {
                 "label": self.Status.COMPLETE,
                 "date": self.computed_end_date,
                 "is_active": self.is_complete,
+                "days": self.computed_duration_days,
             },
         ]
         if not self.is_rollout:
             timeline_entries.insert(
                 4,
                 {
-                    "label": NimbusConstants.ENROLLMENT_END,
+                    "label": NimbusConstants.OBSERVING,
                     "date": self._enrollment_end_date,
-                    "is_active": self._enrollment_end_date is not None,
+                    "is_active": self.is_observing,
+                    "days": self.computed_enrollment_days,
```

Comment:
I think this should be a new computed duration (something like `computed_observation_days = end_date - enrollment_end_date`), and `computed_enrollment_days` should go with the "Enrolling" section (formerly "Live").

Reply:
I believe this is what `computend_enrollment_days` are doing- https://github.com/mozilla/experimenter/blob/54ad0a197b2d3db07ed89e173e89f0342860f5bb/experimenter/experimenter/experiments/models.py#L747

Reply:
`computed_enrollment_days` is slightly different -- it's `enrollment_end_date - enrollment_start_date`, or the period between when experiment became live (enrollment start date) and end of enrollment (enrollment end date). I'm talking about having a different duration that is the number of days between the end of enrollment and the end of the experiment.